config/samples/canary.yaml

```diff
@@ -27,12 +27,12 @@ spec:
       apiVersion: apps/v1beta2
       kind: Deployment
       metadata:
-        name: {{NAME}}
+        name: {{ .Name }}
         labels:
           release: stable
           app:  appinfo
       spec:
-        replicas: {{REPLICAS}}
+        replicas: {{ .Replicas }}
```

Comment:
Why is this not `{{ .Params.Replicas }}`?

Reply:
bc find and replace fail